web/yo/app/views/gene.html

```diff
@@ -365,13 +361,13 @@ <h3 ng-if="reviewMode">Gene Type:
                                  <e-status ng-click="commentClick($event)" ng-keypress="stopCollopse($event)" object="treatment.name_eStatus" id="mutation.name.text + tumor.name.text + ti.name.text + treatment.text" apply-obsolete="applyObsolete(treatment.name_eStatus, 'treatment', mutation, tumor, ti, treatment)"></e-status>
                                 <i ng-click="remove($event, 'treatment', mutation, tumor, ti, treatment)" class="fa fa-trash-o"></i>
                               </span>
-                              <div ng-if="treatment.name_review.get('review') !== 'TREATMENT_NAME_CHANGE'">
+                              <div ng-if="!treNameChangeOnly">
```

Comment:
This variable is a global variable inside of gene.js. Under this scenario, if one treatment name changed for one mutation. All treatments will be shown in the review mode. Right?